drivers/source/usb/TaskBase.cpp

```diff
@@ -21,19 +21,19 @@
 #include "rtos/Semaphore.h"
 #include "platform/mbed_critical.h"
 
-TaskBase::TaskBase(TaskQueue *q)
+events::TaskBase::TaskBase(TaskQueue *q)
```

Comment:
just styling, should this be rather entirely in `namespace events {` `}` rather than each method?